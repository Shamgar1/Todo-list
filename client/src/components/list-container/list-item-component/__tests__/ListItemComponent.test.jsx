import renderer from "react-test-renderer";
import listItemComponent from "../ListItemComponent";

describe("render snapshots", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<listItemComponent />).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it("renders correctly with name", () => {
		const tree = renderer
			.create(<listItemComponent name={"blabla"} />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});

	it("renders correctly with name", () => {
		const tree = renderer
			.create(<listItemComponent name={"blabla"} />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
