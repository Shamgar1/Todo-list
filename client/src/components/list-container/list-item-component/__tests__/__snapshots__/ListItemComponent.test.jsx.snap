// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render snapshots renders correctly 1`] = `<listItemComponent />`;

exports[`render snapshots renders correctly with name 1`] = `
<listItemComponent
  name="blabla"
/>
`;

exports[`render snapshots renders correctly with name 2`] = `
<listItemComponent
  name="blabla"
/>
`;
